### PR TITLE
Gatsby/feature/basic situations

### PR DESCRIPTION
**Pull request by vitkolos**

### CODE REVIEW

File: gatsby/src/components/link/link.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import classnames from 'classnames';
 import { Link as TranslateLink } from 'gatsby-plugin-react-i18next';
 
 interface Props {
-  label: string;
+  label?: string;
```
Comment (fidelman) [inline gatsby/src/components/link/link.tsx:6]: why?

File: gatsby/src/components/list-card/list-card.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Breadcrumb from '@/components/breadcrumb';
+import Link from '@/components/link';
+
+interface IProps {
```
Comment (fidelman) [inline gatsby/src/components/list-card/list-card.tsx:5]: why are they all optional?

File: gatsby/src/templates/situation/list.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+interface IProps {
+  data: IQueryResult;
+}
+
+const Home: React.FC<IProps> = ({ data }) => {
+  const { area } = data;
+  return (
+    <>
```
Comment (fidelman) [inline gatsby/src/templates/situation/list.tsx:19]: not necessary Fragment

File: gatsby/src/templates/situation/list.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      <Container>
+        <Headline>{area.name}</Headline>
+        <div>
+          {area.relationships.situation &&
+            area.relationships.situation.map((item, index) => {
```
Comment (fidelman) [inline gatsby/src/templates/situation/list.tsx:24]:
```suggestion
            area.relationships.situation?.map((item, index) => {
```
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html

File: gatsby/src/templates/situation/page.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import { graphql } from 'gatsby';
+import { IQuery } from 'graphql-types';
+
+interface IProps {
+  data: IQuery;
+}
+
+const Home: React.FC<IProps> = ({ data }) => {
+  return (
+    <>
```
Comment (fidelman) [inline gatsby/src/templates/situation/page.tsx:11]: not necessary Fragment

**Review by fidelman — COMMENTED** [review]
great job 👍 left a few comments

File: gatsby/src/components/link/link.tsx (hunk 1 above)
Comment (kkoudelka) [reply gatsby/src/components/link/link.tsx:6]: You can set content as children of component `<Link>... </Link>`

**Review by kkoudelka — APPROVED** [review]

**Pull request merged**